src/Index/Index.scss

```
@@ -15,6 +15,10 @@
   border-radius: 6px;
 }
 
+.hide {
```

There's no need for a class here, you can do something like this, which is quite simpler
```suggestion
:host([collapsed]) slot {
```